daemon/exec.go

```diff
@@ -210,15 +210,19 @@ func (d *Daemon) ContainerExecStart(ctx context.Context, name string, stdin io.R
 		return err
 	}
 
-	attachConfig := &stream.AttachConfig{
```

Comment:
Removal of the '&' on this line means that it can be put in the argument of the AttachStreams() function call subsequently, which is a tiny hint to any reader that the struct is mutated by that function.

I can switch this back if preferred, but I thought it was better this way around.